src/rust/src/x509/verify.rs

```diff
@@ -0,0 +1,34 @@
+// This file is dual licensed under the terms of the Apache License, Version
+// 2.0, and the BSD License. See the LICENSE file in the root of this repository
+// for complete details.
+
+use crate::x509::certificate::Certificate as PyCertificate;
+
+#[pyo3::pyclass(
+    frozen,
+    name = "Store",
+    module = "cryptography.hazmat.bindings._rust.x509"
+)]
+struct PyStore(Vec<pyo3::Py<PyCertificate>>);
+
+#[pyo3::pymethods]
+impl PyStore {
+    #[new]
+    fn new(certs: &pyo3::types::PyList) -> pyo3::PyResult<Self> {
```

Comment:
I think you can just make this take a `Vec<pyo3::Py<PyCertificate>>)` and it'll DTRT?

Reply:
Yep, was way overthinking it. That works!